microcosm_metrics/naming.py

```diff
@@ -16,4 +16,5 @@ def name_for(*keys, **kwargs):
 
     """
     prefix = kwargs.get("prefix", "microcosm")
-    return ".".join([prefix] + list(keys))
+    environment = kwargs.get("environment", "missing")
+    return ".".join([prefix, environment] + list(keys))
```

Comment:
Would it make sense to group from most-general-to-least?

```
environment > service > route
```

Reply:
It does make sense.

We chose this format because it follows our namespace convention elsewhere. What do you prefer?

Reply:
Elsewhere where?

Reply:
A la the URLs - `oscar.ENV.globality.io` ...

Just makes it more intuitive to navigate the metrics in Datadog - I don't have a strong opinion on this however.